tests/test_commands.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+
+def test_inquire(mapdl):
```

Comment:
This will not work when MAPDL is in server mode as it needs to access the "local" environment. Move `test_inquire` to `test_mapdl.py` and then skip when testing on the "cloud".

Reply:
Done!